.github/workflows/testing.yml

```diff
@@ -33,6 +33,8 @@ jobs:
 
       - name: Setup Kube in Docker
         uses: engineerd/setup-kind@v0.4.0
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: true
 
       - name: Setup latest Kangal release
         run: |
```

Comment:
https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#get-the-latest-release